.changeset/honest-glasses-drum.md

```diff
@@ -0,0 +1,6 @@
+---
+'@openfn/language-salesforce': patch
+---
+
+- add `query()` limits
+- improve `bulk()` and `bulkQuery()` function
```

Comment:
How have we improved these functions? If AK read these release notes, would she understand the benefits of upgrading to this adaptor version?

Reply:
i just removed the `improve bulk() and bulkQuery() function` comment